client/src/utils/csv_parser.ts

```diff
@@ -0,0 +1,158 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {parse} from '../../../node_modules/csv-parse';
```

Comment:
you shouldn't have to reference a package like this. Does `import {parse} from 'csv-parse'` not work?